dist/web3-light.js

```diff
@@ -3327,6 +3348,145 @@ module.exports = ICAP;
     You should have received a copy of the GNU Lesser General Public License
     along with ethereum.js.  If not, see <http://www.gnu.org/licenses/>.
 */
+/** @file ipcprovider.js
+ * @authors:
+ *   Fabian Vogelsteller <fabian@ethdev.com>
+ * @date 2015
+ */
+
+"use strict";
+
+var utils = require('../utils/utils');
+var errors = require('./errors');
+
+
+var IpcProvider = function (path, net) {
+    var _this = this;
+    this.responseCallbacks = {};
+    this.path = path;
+    
+    net = net || require('net');
+
+
+    try {
+        this.connection = net.connect({path: this.path});
+    } catch(error) {
+        throw errors.InvalidConnection(path);
```

Comment:
Its unclear, because some errors are watched by electron before i can catch them and i have to figure out a way to make that reliable and presentable to the user..